File: ppg_sl/csharp/enjou.cs
//言語：cpp
//問題URL：https://paiza.jp/works/mondai/ppg_sl_cs/ppg_sl_cs__enjou
//コメント：
//コード：
//```cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int due = Convert.ToInt32(Console.ReadLine());
        if (due<=3)
        {
            Console.WriteLine("BOMB");
            Environment.Exit(0);
        }
        Console.WriteLine("ONGOING");
    }
}
//```

File: ppg_sl/cpp/enjou.cpp
//言語：cpp
//問題URL：https://paiza.jp/works/mondai/ppg_sl_cpp/ppg_sl_cpp__enjou
//コメント：
//コード：
//```cpp
#include <iostream>

int main() {
    int due;
    std::cin >> due;
    if (due<=3) {
        std::cout << "BOMB" << std::endl;
        exit(0);
    }
    std::cout << "ONGOING" << std::endl;

    return 0;
}
//```